prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = "Johnny Baltazar";
+    console.log(fullName);
+    var numberOfPets = "1";
+    console.log(numberOfPets);
+    var likesVegetables = true; Boolean(true);
+    console.log(likesVegetables);
+    var nothing = "null";
```

Comment:
Same here. `null` is its own value, so we want to omit the quotation marks or JavaScript will think we mean the letters `"n"` `"u"` `"l"` `"l"` instead of the values `null`.